api/src/main/resources/application.yml

```diff
@@ -57,6 +57,7 @@ message-processor:
   # for an explanation of the ISO-8601 duration format
   minimum-lock-time: "PT1M"
   maximum-lock-time: "PT60M"
+  message-limit: 100
```

Comment:
I think we may want to make this a bit bigger than 100. Shall we try 1000 and see how fast we can process them? We probably want about as many as we can process in a minute.

Reply:
Changed to 1000.